client/payment-details/dispute-details/index.tsx

```diff
@@ -55,6 +56,9 @@ const DisputeDetails: React.FC< DisputeDetailsProps > = ( { dispute } ) => {
 									dispute={ dispute }
 									daysRemaining={ countdownDays }
 								/>
+								<EvidenceList
+									issuerEvidence={ dispute.issuer_evidence }
+								/>
```

Comment:
This needs to be placed somewhere better, have just put it here for easier testing.